
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
firstjoint_position_controller:
  type: effort_controllers/JointPositionController
  joint: fixed_torso_to_shoulder
  pid: {p: 2000.0, i: 100, d: 500.0}
secondjoint_position_controller:
  type: effort_controllers/JointPositionController
  joint: shoulder_to_lowerarm
  pid: {p: 50000.0, i: 100, d: 2000.0}
thirdjoint_position_controller:
  type: effort_controllers/JointPositionController
  joint: lowerarm_to_upperarm
  pid: {p: 20000.0, i: 50, d: 1000.0}
fourthjoint_position_controller:
  type: effort_controllers/JointPositionController
  joint: upperarm_to_wrist
  pid: {p: 2000.0, i: 50, d: 200.0}
fifthjoint_position_controller:
  type: effort_controllers/JointPositionController
  joint: wrist_to_finger
  pid: {p: 700.0, i: 50, d: 70.0}